rpm_s3_mirror/config.py

```diff
@@ -63,7 +65,7 @@ def _populate_required(self):
                     raise ConfigError(f"Missing required environment variable: {key.upper()}")
                 else:
                     continue
-            elif key == "upstream_repositories":
+            elif key in ("upstream_repositories", "trim_updates_to_arches"):
```

Comment:
nit: why not use a set?

Reply:
No reason, but doesn't make any difference with two items.